### PR TITLE
Misc: fix up spelling mistake

### PR DESCRIPTION
**Pull request by Jianlin-lv**
Signed-off-by: Jianlin Lv <Jianlin.Lv@arm.com>

### CODE REVIEW

**Review by pchaigno — APPROVED** [review]

File: pkg/datapath/linux/node.go [hunk 1]
```diff
@@ -571,19 +571,19 @@ func (n *linuxNodeHandler) insertNeighbor(newNode *nodeTypes.Node, ifaceName str
 
 	iface, err := net.InterfaceByName(ifaceName)
 	if err != nil {
-		neighborLog("insertNeightbor InterfaceByName", ifaceName, err, &ciliumIPv4, &hwAddr, link)
+		neighborLog("insertNeighbor InterfaceByName", ifaceName, err, &ciliumIPv4, &hwAddr, link)
```
Comment (sayboras) [inline pkg/datapath/linux/node.go]: :100: 

nits: I feel it's kind of repetitive to mention inserNeighbor here as underlying _neighborLog_ is already log this text. https://github.com/cilium/cilium/blob/master/pkg/datapath/linux/node.go#L546

Comment (tklauser) [reply pkg/datapath/linux/node.go]: Good point. I think we should just drop the `insertNeighbor` part from the messages passed to `neighborLog`.

**Review by tklauser — CHANGES_REQUESTED** [review]
As mentioned by @sayboras I think we're just duplicating the `insertNeighbor` string in the log. Better just drop it.

File: pkg/datapath/linux/node.go (hunk 1 above)
Comment (Jianlin-lv) [reply pkg/datapath/linux/node.go]: Agree with you, good comments, Thanks.

**Review by tklauser — APPROVED** [review]
Thanks

**Comment by tklauser** [comment]
Smoke test and Travis CI passed. Since this is only changing log strings, I'll mark this as ready to merge.

**Pull request merged**